test_modules/hello.wren

```diff
@@ -0,0 +1,5 @@
+class Hello {
```

Comment:
If we want to include files like this specifically for testing, it would be better to create a folder called [`testdata`](https://golang.org/cmd/go/#hdr-Test_packages) for them, since they are ignored by the Go tool. Something like `testdata/modules/hello.wren` would be a good alternative for this test, and then the module directory would be set to `testdata/modules`.